app/js/components/system/systemModal/systemModal.jade

```diff
@@ -3,6 +3,16 @@
   .modal-actions
     .fa.fa-close.pull-right(ng-click='close()')
   system-metadata(system='system')
-  .rule-summaries(system='system', machine-id='system.system_id', rule='rule', rule-filter='ruleFilter', loading='loading')
-  .rule-summary.system-overlay-item.content-block(ng-if='system.__fake', report='report', rule='rule', loading='loading')
+  .message.alert.alert-danger.alert-systems-not-checking(ng-if='!system.isCheckingIn')
```

Comment:
This brings back the system-not-checking-in bar which is already on the modal (above). We should only have one.